src/vtt-segment-loader.js

```diff
@@ -279,6 +279,16 @@ export default class VTTSegmentLoader extends SegmentLoader {
 
     this.mediaSecondsLoaded += segment.duration;
 
+  /**
+   * The below code helps in removing double captions
```

Comment:
For comments like this, it's probably better to put it right where the action is taking place, and you don't have to specify that the code is below. Maybe something like:

```
// remove any overlapping cues to prevent doubling
this.remove(firstCueEndTime, lastCueEndTime);
```